contracts/VenusChainlinkOracle.sol

```diff
@@ -92,16 +90,32 @@ contract VenusChainlinkOracle is PriceOracle {
         prices[asset] = price;
     }
 
-    function setFeed(string calldata symbol, address feed) external onlyAdmin() {
+    function batchSetFeeds(string[] calldata symbols_, address[] calldata feeds_, uint[] calldata maxStalePeriods_) external onlyAdmin() {
+        require(symbols_.length == feeds_.length, "invalid length");
+        require(symbols_.length == maxStalePeriods_.length, "invalid length");
+        require(symbols_.length > 0, "empty feeds");
+        for (uint i = 0; i < symbols_.length; i++) {
+            setFeed(symbols_[i], feeds_[i], maxStalePeriods_[i]);
+        }
+    }
+
+    function setFeed(string memory symbol, address feed, uint maxStalePeriod) public onlyAdmin() {
```

Comment:
just wanted to leave an interface to support setting a single feed, do you mean we get rid of this and only leave batch setting?

Reply:
No, I propose we have a separate internal function so that we could call it from both the batch version and the single version. This way we could avoid uplicate `onlyAdmin` checks:

```suggestion
    function setFeed(string memory symbol, address feed, uint maxStalePeriod) external onlyAdmin() {
        setFeedInternal(symbol, feed, maxStalePeriod);
    }
    
    function setFeedInternal(string memory symbol, address feed, uint maxStalePeriod) internal {
        // ...
```